chain/bitcoind.go

```diff
@@ -919,6 +930,22 @@ func (c *BitcoindClient) rescan(hash *chainhash.Hash) error {
 		if i%10000 == 0 {
 			c.onRescanProgress(lastHash, i, block.Header.Timestamp)
 		}
+
+		if i == bestBlock.Height {
```

Comment:
Why was this needed? Could use a comment explaining rationale.

Reply:
Added comment.